### ISSUE TITLE
Quasar not working

### ISSUE THREAD

**Issue by Sweazou** [quasar/Quasar#961]
Hi,
i can't receive connections from my hosts. Everything seem fine with the setup, firewall and ports.

![Client](https://user-images.githubusercontent.com/50371041/115168949-ab9c2480-a08a-11eb-83b3-2b976ede4594.png)
![Settings](https://user-images.githubusercontent.com/50371041/115168954-adfe7e80-a08a-11eb-95b8-239ece00a562.png)
![Firewall](https://user-images.githubusercontent.com/50371041/115168960-b1920580-a08a-11eb-868a-3f1b35f82da0.png)
![Port](https://user-images.githubusercontent.com/50371041/115168961-b35bc900-a08a-11eb-94aa-8dd26388d333.png)

**Comment by dsfdsdwcdsfsdf** [comment]
i also have the same issue, i have the startup process enabled, but when i restarted my pc the connection disappeared and wont come back online

**Comment by ErickDaGamer** [comment]
Are you sure that you port forwarded correctly?

**Comment by BurntDog** [comment]
Use https://canyouseeme.org/ to make sure port is seen/open and not blocked by ISP.

**Comment by Sweazou** [comment]
> Are you sure that you port forwarded correctly?

Yes

**Comment by Sweazou** [comment]
> Use https://canyouseeme.org/ to make sure port is seen/open and not blocked by your ISP.

There is already a screen showing it.

**Comment by BurntDog** [comment]
Sorry missed that.

**Comment by BurntDog** [comment]
Also you did a test loop back and all is good?

**Comment by Sweazou** [comment]
Yes

**Comment by BurntDog** [comment]
Are you able to test client on a different computer? Also you can confirm client is running? Did Windows defender or other AV delete the file? Did you put the client file and or location of, in your AV's exclusion list?

**Comment by Sweazou** [comment]
Yes the client is running on the second laptop and excluded from av and i still don't see it on my main computer.

**Comment by BurntDog** [comment]
What if you temporarily turn off both firewalls to test and see if it connects then?

**Comment by Sweazou** [comment]
Yeah i already tried that and it still doesn't work ...

**Comment by OusmBlueNinja** [comment]
i have the same problem the 172.0.0.1 loobback works and 192.168.0.10 (my lan ip) works but when i put in my portmap.io or public ip it doesent connect ive lookaed at my firewall and my clients firewall and its not blocking anything the port is not block and is being listend to.   pls help ask for screenshots if u need them ive been tring to fix it for 3 days

**Comment by OusmBlueNinja** [comment]
i have the same problem the 172.0.0.1 loobback works and 192.168.0.10 (my lan ip) works but when i put in my portmap.io or public ip it doesent connect ive lookaed at my firewall and my clients firewall and its not blocking anything the port is not block and is being listend to.   pls help ask for screenshots if u need them ive been tring to fix it for 3 days

**Comment by BurntDog** [comment]
There's a video on YouTube to help with "portmap" and you can't use your public ip as a loopback it won't work. I do believe portmap address expires after a period of time if not used, is your portmap settings new/within a week or so? This issues has been resolved numerous time you can do a search in issues to find a solution.

**Comment by OusmBlueNinja** [comment]
sorry i got it figured out im useing open VPN now with some random ip cus portamap sux

**Comment by OusmBlueNinja** [comment]
so i tested it and it doesent work on outside connections

**Comment by OusmBlueNinja** [comment]
and when o do 172.0.0.1 it works so its just not showing up on the list of pple

**Comment by DoCmAL1** [comment]
No ip is blocking connections involving alleged "rat" traffic. Also no ip and tons of other dns just log your information.

**Comment by BurntDog** [comment]
Just checked and portmap.io is working just fine. The thing with portmap.io is you have to make a new mapping rule cause the free account auto deletes it after a period of time.

**Comment by flashbackboi** [comment]
> Use https://canyouseeme.org/ to make sure port is seen/open and not blocked by your ISP.

if blocked then?

**Comment by Sweazou** [comment]
> > Use https://canyouseeme.org/ to make sure port is seen/open and not blocked by your ISP.
> 
> if blocked then?
> 

Open your ports

**Comment by flashbackboi** [comment]
> > > Use https://canyouseeme.org/ to make sure port is seen/open and not blocked by your ISP.
> > 
> > 
> > if blocked then?
> 
> Open your ports

how tf
